### PR TITLE
add documentation regarding volumes and claims

### PR DESCRIPTION
**Pull request by linki**
This PR describes how to use a `persistentVolumeClaim` together with a `Deployment`.

Not covered
* using `volumeClaimTemplates` with `StatefulSets`

### CODE REVIEW

**Comment by hjacobs** [comment]
@linki maybe using PostgreSQL as an example might be confusing as we want to promote/encourage Spilo for PostgreSQL.

**Comment by linki** [comment]
@hjacobs yeah, could be. I'll switch it with something else. any recommendations?

**Comment by hjacobs** [comment]
@linki maybe Redis?

**Comment by linki** [comment]
changed the example from postgres to redis

**Comment by hjacobs** [comment]
Creating a public ELB is probably a bad idea for Redis as it does not provide real authentication.

Can we simply change it to use `ClusterIP` service and `kubectl run ...` to connect to it (also has the advantage that users don't need to install `redis-cli` locally)?

**Comment by linki** [comment]
changed to using clusterIP instead of exposing the service

**Comment by hjacobs** [comment]
:+1:

File: docs/user-guide/using-volumes.rst [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+Find out where your pod currently runs:
+
+.. code-block:: bash
+
+    $ kubectl get pods -o wide
+      NAME                        READY     STATUS    RESTARTS   AGE       IP          NODE
+      redis-3548935762-qevsk      1/1       Running   0          2m        10.2.1.66   ip-172-31-15-65.eu-central-1.compute.internal
+
+The node it landed on is ``ip-172-31-15-65.eu-central-1.compute.internal``. Connect to your exposed redis endpoint and create a table:
```
Comment (mikkeloscar) [inline docs/user-guide/using-volumes.rst]: > Connect to your exposed redis endpoint...

I guess this should be removed.

Comment (linki) [inline docs/user-guide/using-volumes.rst]: changed it accordingly

**Comment by hjacobs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**